shared/fs/container.js

```diff
@@ -78,9 +81,14 @@ class ChooseComponent extends React.PureComponent<ChooseComponentProps> {
     if (this.props.path !== prevProps.path) {
       this.props.loadPathMetadata()
     }
+    if (this.props.kbfsDaemonStatus !== 'connected') {
```

Comment:
we should discuss. i dislike components driving something like this. also i dislike a thing that could get called a million times by accident like this can

Reply:
This doesn’t always results in a RPC call; saga part deduplicates. I just wanted a way to trigger the waitForClient call again after timeout first time. It seems this would result in less calls since it never calls as long as user doesn’t go into the Files tab.

Would hooking this up with routing be better here?

Reply:
@chrisnojima bump!